drivers/sensor/st/lsm6dsv16x/lsm6dsv16x_trigger.c

```diff
@@ -26,7 +26,7 @@ static int lsm6dsv16x_enable_xl_int(const struct device *dev, int enable)
 {
 	const struct lsm6dsv16x_config *cfg = dev->config;
 	stmdev_ctx_t *ctx = (stmdev_ctx_t *)&cfg->ctx;
-	int ret;
+	int ret = 0;
```

Comment:
This change is not required, since we are not reading this value before assigning something to it.

Reply:
I think I changed it, but that doesn't matter. Let's just keep it, shall we?

Reply:
I would revert it, coz imho it's unrelated to the issue afaik.